src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -50,8 +50,7 @@
     private StateMachineAdapter stateMachineAdapter;
 
     public CaseDTO getCase(Long suitId, Long caseId) {
-        Suit suit = suitDAO.findOne(suitId);
-        checkNotNull(suit);
+        Suit suit = suitTransformer.fromDto(suitService.getSuit(suitId));
```

Comment:
suitService converts suit to suitDto, then caseService transforms it back to suit. This behaviour should be avoided.

Reply:
However, I fixed this behaviour.
Done